website/docs/r/log_parsing_rule.html.markdown

```diff
@@ -0,0 +1,80 @@
+---
+layout: "newrelic"
+page_title: "New Relic: newrelic_log_parsing_rule"
+sidebar_current: "docs-newrelic-resource-log-parsing-rule"
+description: |-
+Create and manage Log Parsing Rule.
+---
+
+# Resource: newrelic\_log\_parsing\_rule
+
+Use this resource to create, update and delete New Relic Log Parsing Rule.
+
+## Example Usage
+
+Use this example to create the log parse rule.
+```hcl
+
+resource "newrelic_log_parsing_rule" "foo"{
+    account_id  = 12345
+    name        = "log_parse_rule"
+    attribute   = "message"
+    enabled     = true
+    grok        = "sampleattribute='%%{NUMBER:test:int}'"
+    lucene      = "logtype:linux_messages"
+    nrql        = "SELECT * FROM Log WHERE logtype = 'linux_messages'"
+}
+
+```
+
+## Additional Example
+Use this example to validate a grok pattern and create the log parse rule.  More
+information on grok pattern can be found [here](https://docs.newrelic.com/docs/logs/ui-data/parsing/#grok)
+```hcl
+data "newrelic_test_grok_pattern" "grok"{
+    account_id  = 12345
```

Comment:
Same comment here, remove the account_id

Reply:
removed account Id